display/d.linegraph/main.c

```diff
@@ -182,7 +182,7 @@ int main(int argc, char **argv)
         FILE *fp;             /* pointer to file        */
     };
 
-    struct in_file in[12];
+    struct in_file in[12] = {0};
```

Comment:
I guess you started the new branch from the branch you used for the other PR. Switch to main first.

Reply:
actually i used an existing branch. From next time i will create separate branch